### DISCUSSION TITLE
Hi, is that possbile to config a wildcard record for a domain ?

### DISCUSSION THREAD

**Discussion by WestFarmer** [TechnitiumSoftware/DnsServer#788] [Q&A]
I don't known too much of dns.

I have a domain name `itps.xxx.com`

because we are using istio to expose services in k8s cluster.

the istio ingress gateway is the only entry point for all services.

from example:

```
a.itps.xxx.com
b.itps.xxx.com
c.itps.xxx.com
d.itps.xxx.com
e.itps.xxx.com
```

all should be pointed to the ELB address of  istio ingress gateway

everytime we add a new service, we need to add one dns record.

I am wondering, is there something in dns can work like this:
```
record 1:  exceptional.itps.xxx.com  -> IP1
record 2: *.itps.xxx.com -> IP2
```

**Comment by ShreyasZare** [comment]
Thanks for asking. Yes, you just have to add record with `*` and it will work.
